tests/components/azure_storage/test_backup.py

```diff
@@ -38,6 +39,7 @@ async def setup_backup_integration(
         patch("homeassistant.components.backup.is_hassio", return_value=False),
         patch("homeassistant.components.backup.store.STORE_DELAY_SAVE", 0),
     ):
+        async_initialize_backup(hass)
```

Comment:
I don't like this fragmented setup requirement.

Reply:
Yes, it's terrible.

The root issue is that frontend has no way to wait for core integrations to be ready, and we generally work around that by adding integrations which has APIs frontend needs to frontend's `after_dependencies` or `dependencies`. However, that has unwanted consequences because frontend is set up very early.

Going forward, we should add a way for frontend to check which integrations will be setup and a way for it to wait for an integration to set up.